tensorflow-app-dependencies/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.springframework.cloud.stream.app</groupId>
+    <artifactId>tensorflow-app-dependencies</artifactId>
+    <version>1.2.1.BUILD-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>tensorflow-app-dependencies</name>
+    <description>Spring Cloud Stream tensorflow App Dependencies</description>
+
+    <parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.2.1.RELEASE</version>
+        <relativePath />
+    </parent>
+
+    <properties>
+        <pmml.version>1.2.6</pmml.version>
+		<tensorflow.version>1.1.0-rc2</tensorflow.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-bus</artifactId>
+				<version>1.3.0.RELEASE</version>
+			</dependency>
+			<dependency>
+                <groupId>org.springframework.cloud.stream.app</groupId>
+                <artifactId>spring-cloud-starter-stream-processor-tensorflow</artifactId>
+                <version>1.2.1.BUILD-SNAPSHOT</version>
+            </dependency>
+			<dependency>
+				<groupId>org.springframework.cloud.stream.app</groupId>
+				<artifactId>spring-cloud-starter-stream-processor-twitter-sentiment</artifactId>
+				<version>1.2.1.BUILD-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.tensorflow</groupId>
+				<artifactId>tensorflow</artifactId>
+				<version>${tensorflow.version}</version>
+			</dependency>
+			<dependency>
+				<groupId>commons-io</groupId>
+				<artifactId>commons-io</artifactId>
```

Comment:
acknowledged. After replacing the stream to byte utility with StreamUtils this deps is not required anymore and will be removed